packages/docs/components/ColorCards/ColorCard.tsx

```diff
@@ -0,0 +1,83 @@
+import { Flex, Link, Small, Text } from '@bigcommerce/big-design';
+import React, { useContext } from 'react';
+import styled, { ThemeContext } from 'styled-components';
+
+import { ColorDetails } from './availableColors';
+
+const StyledColor = styled(Flex)`
+  height: ${({ theme }) => theme.helpers.remCalc(160)};
+  width: ${({ theme }) => theme.helpers.remCalc(184)};
+  border-radius: 4px;
+`;
+
+interface ColorCardProps {
+  colorCard: ColorDetails;
+}
+
+const ColorCard: React.FC<ColorCardProps> = ({ colorCard }) => {
+  const { colors } = useContext(ThemeContext);
+  const hasDescription = colorCard.description;
+  const hasHexagesimalColor = colors[colorCard.name].startsWith('#');
```

Comment:
🍹  `hasHexColor`

Reply:
🍹  We can use a `const colorValue = colors[colorCard.name]` since we use it in multiple places.